content/02.intro.md

```diff
@@ -8,5 +8,6 @@ As DL is an active and specialized research area, detailed resources are rapidly
 To address this issue, we solicited input from a community of researchers with varied biological and deep learning interests, who wrote this manuscript collaboratively using the GitHub version control platform [@url:https://github.com/Benjamin-Lee/deep-rules] and Manubot [@url:https://greenelab.github.io/meta-review/].
 
 In the course of our discussions, several themes became clear: the importance of understanding and applying ML fundamentals [@doi:10.1186/s13040-017-0155-3] as a baseline for utilizing DL, the necessity for extensive model comparisons with careful evaluation, and the need for critical thought in interpreting results generated by means of DL, among others.
+Further, since both DL models and the datasets to which they are applied impact prediction results, it is important to consider [NOUN] when generating biological or clinical insights from these *in silico* methods.
```

Comment:
```suggestion
Further, since both DL models and the datasets to which they are applied impact prediction results, it is important to consider both when generating biological or clinical insights from these *in silico* methods.
```

Reply:
@Benjamin-Lee - want to take a look at this? I think this was probably what was intended.

Reply:
I'm going to commit this suggestion so I can also handle the merge.